src/main/kotlin/open/tresorier/services/OperationService.kt

```diff
@@ -123,7 +129,27 @@ class OperationService(private val operationDao: IOperationDao, private val auth
         if (type == "DEBIT") {
             amount *= -1
         }
-        val operationCreated = Operation(account.id, day, null, amount,Time.now(), memo, false, false)    // créer une opération sans la mettre dans la base de donnée
+        val operationCreated = Operation(account.id, day, null, amount,Time.now(), memo, false, false, null)
         return operationCreated
     }
+    fun findDaughterOperations(person: Person, motherOperation: Operation): List<Operation> {
+        val account = accountDao.getById(motherOperation.accountId)
+        authorizationService.cancelIfUserIsUnauthorized(person, account)
+        val listDaughterOperation = operationDao.findDaughterOperations(motherOperation)
+        return listDaughterOperation
+    }
+    fun findMotherOperationsByAccount(person: Person, account: Account, category: Category?) : List<Operation> {
+        authorizationService.cancelIfUserIsUnauthorized(person, account)
+        var listMotherOperations = operationDao.findMotherOperationsByAccount(account, category)
+        return listMotherOperations
```

Comment:
detail but you don't need a variable, you can just return the result

Reply:
ok